src/components/OrganizationListTable.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Organization, OrganizationBundle } from '@pdc/sdk';
+import {
+	Table,
+	TableHead,
+	ColumnHead,
+	TableBody,
+	TableRow,
+	RowCell,
+} from './Table';
+
+interface OrganizationListTableRowProps {
+	organization: Organization;
+}
+
+const OrganizationListTableRow = ({
+	organization,
+}: OrganizationListTableRowProps) => {
+	const navigate = useNavigate();
+
+	const handleRowClick = () => {
+		// Only navigate if we didn't just finish selecting text.
+		if (window.getSelection()?.isCollapsed) {
+			navigate(`/organizations/${organization.id}`);
+		}
+	};
+	return (
+		<TableRow onClick={handleRowClick}>
+			<RowCell>{organization.name}</RowCell>
+			<RowCell>{organization.employerIdentificationNumber}</RowCell>
+		</TableRow>
+	);
+};
+
+interface OrganizationListTableProps {
+	columns: string[];
+	organizations: OrganizationBundle;
+}
+
+export const OrganizationListTable = ({
```

Comment:
Please don't boo me off the stage BUT... should we make a generic `ListTable` similar to what we did with the `ListGrid`?  I think this will become more obviously useful when we add base fields to the organizations entity but even now I think there is some rendering / UX logic that is being duplicated.

Reply:
Boo! Boo! No, i'm kidding. This does seem like the right idea, let me see how feasible it is really quickly.